src/stravalib/model.py

```diff
@@ -378,32 +390,6 @@ class DetailedClub(SummaryClub, strava_model.DetailedClub):
     pass
 
 
-class Gear(DetailedGear):
-    """
-    Represents a piece of gear (equipment) used in physical activities.
-    """
-
-    pass
-
-
-class Bike(Gear):
-    """
-    Represents a bike as a "type" / using the structure of
-    `stravalib.model.Gear`.
-    """
-
-    pass
-
-
-class Shoe(Gear):
-    """
-    Represents a Shoes as a "type" / using the structure of
-    `stravalib.model.Gear`.
-    """
-
-    pass
-
-
```

Comment:
We're not adding any behavior to Gear, so let's just refer to `strava_model.(Summary-)Gear`.